MonacoEditorComponent/CodeEditor.cs

```diff
@@ -90,20 +88,6 @@ private void CodeEditor_Loaded(object sender, RoutedEventArgs e)
             {
                 _model = new ModelHelper(this);
 
-                _parentAccessor = new ParentAccessor(this);
```

Comment:
Where did this section get moved to?

Reply:
I've moved this line to WebView_NavigationStarting

Reply:
Ah, cool.  I didn't realize I had it duplicated accidentally.  Thanks!